src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/NotificationRepository.cs

```diff
@@ -62,12 +86,56 @@ public async Task<IReadOnlyCollection<Notification>> GetMemberNotificationsWithM
-                                 FROM Notifications n
-                                 INNER JOIN Messages m ON n.MessageId = m.Id
-                                 INNER JOIN Members me ON m.OwnerId = me.Id
-                                 INNER JOIN Channels c ON n.ChannelId = c.Id
-                                 WHERE n.MemberId = @memberId";
+                var sqlQuery = $@"
+                    SELECT 
+                        n.{nameof(Notification.Id)}, 
+                        n.{nameof(Notification.IsRead)}, 
+                        n.{nameof(Notification.MessageId)}, 
+                        n.{nameof(Notification.ChannelId)}, 
+                        n.{nameof(Notification.MemberId)},
+                        m.{nameof(Message.Id)},
+                        m.{nameof(Message.Body)},
+                        m.{nameof(Message.Created)},
+                        m.{nameof(Message.ImageUrl)},
+                        m.{nameof(Message.Type)},
+                        m.{nameof(Message.ChannelId)},
+                        m.{nameof(Message.OwnerId)},
+                        m.{nameof(Message.Updated)},
+                        m.{nameof(Message.ForwardMessageId)},
+                        m.{nameof(Message.AccessibilityStatus)},
+                        me.{nameof(Member.Id)},
+                        me.{nameof(Member.Email)},
+                        me.{nameof(Member.IsAfk)},
+                        me.{nameof(Member.IsBanned)},
+                        me.{nameof(Member.LastActivity)},
+                        me.{nameof(Member.LastNudged)},
+                        me.{nameof(Member.Name)},
+                        me.{nameof(Member.PhotoName)},
+                        me.{nameof(Member.Role)},
+                        me.{nameof(Member.SaasUserId)},
+                        me.{nameof(Member.Status)},
+                        me.{nameof(Member.IsActive)},
+                        c.{nameof(Channel.Id)}, 
+                        c.{nameof(Channel.Created)}, 
+                        c.{nameof(Channel.Name)}, 
+                        c.{nameof(Channel.CreatorId)}, 
+                        c.{nameof(Channel.IsClosed)}, 
+                        c.{nameof(Channel.MembersCount)}, 
+                        c.{nameof(Channel.Type)}, 
+                        c.{nameof(Channel.Description)}, 
+                        c.{nameof(Channel.WelcomeMessage)}, 
+                        c.{nameof(Channel.Updated)}, 
+                        c.{nameof(Channel.PhotoUrl)}
+                    FROM 
+                        Notifications n
+                    INNER JOIN Messages m 
+                        ON n.{nameof(Notification.MessageId)} = m.{nameof(Message.Id)}
+                    INNER JOIN Members me 
+                        ON m.{nameof(Message.OwnerId)} = me.{nameof(Member.Id)}
+                    INNER JOIN Channels c 
+                        ON n.{nameof(Message.ChannelId)} = c.{nameof(Channel.Id)}
```

Comment:
I suppose that here should be {nameof(Notification.ChannelId)} instead of {nameof(Message.ChannelId)}